diff --git a/robotd/master.py b/robotd/master.py
index d4af9e8..6670f57 100644
--- a/robotd/master.py
+++ b/robotd/master.py
@@ -224,8 +224,7 @@ def main(**kwargs):
         master.cleanup()
 
 
-if __name__ == '__main__':
-
+def main_cmdline():
     # Parse terminal arguments
     import argparse
     parser = argparse.ArgumentParser()
@@ -236,3 +235,7 @@ if __name__ == '__main__':
     args = parser.parse_args()
 
     main(**vars(args))
+
+
+if __name__ == '__main__':
+    main_cmdline()
diff --git a/setup.py b/setup.py
index 3e7ee33..859709c 100644
--- a/setup.py
+++ b/setup.py
@@ -26,5 +26,10 @@ setup(
         'numpy',
         'setproctitle',
     ],
+    entry_points={
+        'console_scripts': [
+            'robotd = robotd.master:main_cmdline',
+        ],
+    },
     zip_safe=False,
 )
